.github/workflows/reusable-web-e2e.yml

```diff
@@ -186,12 +200,25 @@ jobs:
         path: playwright-report
         retention-days: 14
 
+    - name: Notify Slack Action
+      uses: ravsamhq/notify-slack-action@v2
+      if: ${{ always() && inputs.slack_notification }} && ${{ needs.e2e_web.result == 'failure' }}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:**  Why is the `always()` expression ([ref to docs](https://docs.github.com/en/actions/learn-github-actions/expressions#always)) needed here? It seems redundant when the `inputs.slack_notification` is taking effect already‏

Reply:
@rifont if some step of the merge-reports job fails then a job stops on a failed step and we won't receive a notification. Because the merge-reports job is a part of WEB tests we have to get a notification even when this job fails.